src/model_config_utils.cc

```diff
@@ -784,6 +784,31 @@ GetNormalizedModelConfig(
   return Status::Success;
 }
 
+Status
+SetDefaultInstanceCount(
+    ::inference::ModelInstanceGroup* group, const std::string& backend)
+{
+  const int default_cpu_instance_count = 2;
+  if (group->kind() == inference::ModelInstanceGroup::KIND_CPU) {
+    // Backends opt into the default_cpu_instance_count since
+    // some backends don't perform well/have high overhead
```

Comment:
Can you add some examples? Looks like they are OpenVINO and Torch backend?

Reply:
Updated comment to use Pytorch and OpenVINO as examples of backends which don't perform well with multiple instances.